### PR TITLE
Including brew tap instructions

### PR DESCRIPTION
**Pull request by ronny-mysten**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: docs/content/guides/developer/getting-started/sui-install.mdx [hunk 1]
```diff
@@ -41,7 +41,7 @@ Each Sui release provides a set of binaries for several operating systems. You c
 
 ### Confirm the installation {#confirm-the-installation}
 
-To confirm that Sui installed correctly, type `sui` and press Enter. You should see a message about the Sui version installed and help for using Sui commands.
+To confirm that Sui installed correctly, type `sui` and press Enter. You should see a message about the Sui version installed and help for using Sui commands.
```
Comment (stefan-mysten) [inline docs/content/guides/developer/getting-started/sui-install.mdx:44]: Nit: whitespace added at the end.

**Review by stefan-mysten — CHANGES_REQUESTED** [review]
I would recommend we put `Homebrew complete install` first thing on the top page, before install from binaries. It is buried too deep down.

File: docs/content/guides/developer/getting-started/sui-install.mdx [hunk 2]
```diff
@@ -211,24 +211,46 @@ sudo apt-get install build-essential
...
-Use the following command to install [Brew](https://brew.sh/):
+Use the following command to install [Homebrew](https://brew.sh/):
 
 ```bash
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 ```
 
+### Homebrew complete install
+
+You can install Sui along with prequisites using the following commands.
```
Comment (ronny-mysten) [inline docs/content/guides/developer/getting-started/sui-install.mdx:231]:
```suggestion
You can install Sui along with prerequisites using the following commands.
```

File: docs/content/guides/developer/getting-started/sui-install.mdx (hunk 1 above)
Comment (ronny-mysten) [inline docs/content/guides/developer/getting-started/sui-install.mdx:44]:
```suggestion
To confirm that Sui installed correctly, type `sui` and press Enter. You should see a message about the Sui version installed and help for using Sui commands.
```

**Comment by ronny-mysten** [comment]
> Thanks for the super sonic fast update to this :D I would recommend we put `Homebrew complete install` first thing on the top page, before install from binaries. It is buried too deep down.

Not that fast as I had already created the content when the idea was made :) And the problem with putting that high up is that it's only for Mac users. Would also necessitate bringing up the Homebrew install info. Would also mess up the tabbed structure of the instructions.

**Comment by stefan-mysten** [comment]
> > Thanks for the super sonic fast update to this :D I would recommend we put `Homebrew complete install` first thing on the top page, before install from binaries. It is buried too deep down.
> 
> Not that fast as I had already created the content when the idea was made :) And the problem with putting that high up is that it's only for Mac users. Would also necessitate bringing up the Homebrew install info. Would also mess up the tabbed structure of the instructions.

Homebrew can be used also on Linux and Windows, though I have not tried it. Ultimately feel free to decide what's best, but if as a user I have to scroll down through a long page to get to this two line command to install Sui, that's not a great user experience. I still vote for it being on the top, before installing via binaries.
```
# Homebrew install (including all prerequisites)
brew tap mystenlabs/tap
brew install mystenlabs/tap/sui
```

**Review by stefan-mysten — APPROVED** [review]

**Review by johnjmartin — COMMENTED** [review]
Agreed with @stefan-mysten on putting the homebrew install first on the list. It is just much less work to install than the other options, and it's really easy to filter out if you choose not to install via homebrew

**Review by johnjmartin — APPROVED** [review]

**Comment by stefan-mysten** [comment]
Thanks @ronny-mysten !!!

**Pull request merged**